beacon-chain/sync/service.go

```diff
@@ -178,6 +178,7 @@ type Service struct {
 	lcStore                          *lightClient.Store
 	dataColumnLogCh                  chan dataColumnLogEntry
 	registeredNetworkEntry           params.NetworkScheduleEntry
+	spawner                          func(func())
```

Comment:
Is it possible to add a comment here? Neither the name (`spawner`) nor the type (`func(func())`) helps the reader about the utility of this field.

Reply:
Or maybe just renaming it in `subscriptionSpawner`?